cmd/dashboard.go

```diff
@@ -99,6 +102,12 @@ dapr dashboard -k -p 0
 			os.Exit(1)
 		}
 
+		if err := utils.CheckIfPortAvailable(dashboardLocalPort); err != nil {
+			print.FailureStatusEvent(os.Stderr, "Please select a different port with %q flag: %s", "-p", err)
```

Comment:
Will this error message inform the user that the port is busy?

Reply:
Do we need "%q" with "-p", can we use the flag in string directly?

Reply:
> Will this error message inform the user that the port is busy?

yes.

> Do we need "%q" with "-p", can we use the flag in string directly?

yes, we will have to use %q here.